tests/bluetooth/ctrl_isoal/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+include_directories("./src")
```

Comment:
You don't hvae any header files in src.

Is someone #including main.c ?

Reply:
Removed. Done.